recipes/clonalframeml/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: clonalframeml
+  version: "1.11"
+
+build:
+  number: 3
+
+source:
+  fn: ClonalFrameML-1.11.tar.gz
+  url: https://github.com/xavierdidelot/ClonalFrameML/archive/v1.11.tar.gz
+  md5: f137cc4214d71ca1a63e075d052687f3
+
+requirements:
+  build:
+  - gcc # [linux]
+  - llvm # [osx]
+  run:
+  - libgcc # [linux]
+
+test:
+  commands:
+    - ClonalFrameML 2>&1 | grep "ClonalFrameML"
+
+about:
+  home: https://github.com/xavierdidelot/ClonalFrameML
+  license: GPLv3
```

Comment:
Can you provide a `license_file`

Reply:
Thanks for your comments. The authors did not provide the license file but mentioning that the source code follows GPLv3, should I include this one https://www.gnu.org/copyleft/gpl.html ?
Do I have to create new PR?

Reply:
I will discuss a general approach in an issue as including in every recipe the GPL (as hardcopy) also seems wired, we should do better.